### ISSUE TITLE
Uncaught TypeError: Cannot read property 'path' of undefined

### ISSUE THREAD

**Issue by dbperez** [joefitzgerald/go-plus#157]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.180.0
**System**: Mac OS X 10.10.2
**Thrown From**: [go-plus](https://github.com/joefitzgerald/go-plus) package, v3.2.2

### Stack Trace

Uncaught TypeError: Cannot read property 'path' of undefined

```
At /Users/bperez/.atom/packages/go-plus/node_modules/atom-message-panel/lib/LineMessageView.js:51

TypeError: Cannot read property 'path' of undefined
  at [object Object].LineMessageView.goToLine (/Users/bperez/.atom/packages/go-plus/node_modules/atom-message-panel/lib/LineMessageView.js:51:58)
  at HTMLDivElement.<anonymous> (/Users/bperez/.atom/packages/go-plus/node_modules/atom-message-panel/node_modules/space-pen/lib/space-pen.js:214:36)
  at HTMLDivElement.jQuery.event.dispatch (/Users/bperez/.atom/packages/go-plus/node_modules/atom-message-panel/node_modules/space-pen/vendor/jquery.js:4676:9)
  at HTMLDivElement.elemData.handle (/Users/bperez/.atom/packages/go-plus/node_modules/atom-message-panel/node_modules/space-pen/vendor/jquery.js:4360:46)

```

### Commands

```
     -0:44.0 core:paste (atom-text-editor.editor)
     -0:42.8 core:undo (atom-text-editor.editor)
     -0:41.7 editor:newline (atom-text-editor.editor)
     -0:41.4 core:move-up (atom-text-editor.editor)
     -0:41.2 core:paste (atom-text-editor.editor)
     -0:40.8 editor:move-to-first-character-of-line (atom-text-editor.editor)
     -0:40.6 editor:select-to-end-of-word (atom-text-editor.editor)
     -0:39.1 autocomplete-plus:cancel (atom-text-editor.editor)
     -0:38.4 core:move-down (atom-text-editor.editor)
  5x -0:35.9 editor:move-to-end-of-word (atom-text-editor.editor)
  2x -0:34.7 editor:select-to-end-of-word (atom-text-editor.editor)
     -0:33.1 core:backspace (atom-text-editor.editor)
     -0:32.4 core:save (atom-text-editor.editor)
     -0:31.0 core:move-up (atom-text-editor.editor)
     -0:12.0 core:save (atom-text-editor.editor)
  4x -0:10.4 window:save-all (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "refactor"
    ]
  },
  "go-plus": {
    "goPath": "/Users/bperez/development/go",
    "showPanelWhenNoIssuesExist": true,
    "formatTool": "gofmt"
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-plus, v2.1.6
go-plus, v3.2.2
language-docker, v1.1.2
minimap, v4.2.2
project-manager, v1.15.5

# Dev
No dev packages
```

**Comment by vanackere** [comment]
Hi, same bug here with recent versions of atom. See how it was fixed in atom-typescript for example:

https://github.com/TypeStrong/atom-typescript/commit/ac3ca29a8ff1c2c1726a208b93d9d71c87b5726c

**Comment by vanackere** [comment]
By the way, this is easy to reproduce by simply clicking on any compile warning/error in the console (same bug occurs also on linux and windows...)

**Comment by arashpayan** [comment]
Experiencing the same as @vanackere. Clicking any compiler warning in the console triggers this, even after a fresh launch.

**Comment by joefitzgerald** [comment]
This is fixed in the dependent component but it requires an uninstall / install or an apm install in the package directory to resolve. As a result, I will publish a patch release so you don't have to deal with that.

**Comment by joefitzgerald** [comment]
Fixed in 3.2.3.
